<html><head></head><body><ul><li><span> Rossini M, Montanaro G, Montreuil O, Tarasov S., </span><span> "Towards computable taxonomic knowledge: Leveraging nanopublications for sharing new synonyms in the Madagascan genus &lt;i&gt;Helictopleurus&lt;/i&gt; (Coleoptera, Scarabaeinae).", </span><span><b> 12 </b></span><span> (2024): </span><span><i> Biodiversity data journal, </i></span><span> e120304, </span><span> DOI: 10.3897/bdj.12.e120304, </span></li><li><span> Bucur CI, Kuhn T, Ceolin D, van Ossenbruggen J., </span><span> "Nanopublication-based semantic publishing and reviewing: a field study with formalization papers.", </span><span><b> 9 </b></span><span> (2023): </span><span><i> PeerJ. Computer science, </i></span><span> e1159, </span><span> DOI: 10.7717/peerj-cs.1159, </span></li><li><span> Montanaro G, Balhoff JP, Gir&#243;n JC, S&#246;derholm M, Tarasov S., </span><span> "Computable species descriptions and nanopublications: applying ontology-based technologies to dung beetles (Coleoptera, Scarabaeinae).", </span><span><b> 12 </b></span><span> (2024): </span><span><i> Biodiversity data journal, </i></span><span> e121562, </span><span> DOI: 10.3897/bdj.12.e121562, </span></li><li><span> "Peer Review of &#8220;Representing Physician Suicide Claims as Nanopublications: Proof-of-Concept Study Creating Claim Networks&#8221;", </span><span><b> 3(3) </b></span><span> (2022): </span><span><i> JMIRx med, </i></span></li><li><span> "Peer Review of &#8220;Representing Physician Suicide Claims as Nanopublications: Proof-of-Concept Study Creating Claim Networks&#8221;", </span><span><b> 3(3) </b></span><span> (2022): </span><span><i> JMIRx med, </i></span></li><li><span> Soares Teles A., </span><span> "Peer Review of &#8220;Representing Physician Suicide Claims as Nanopublications: Proof-of-Concept Study Creating Claim Networks&#8221;", </span><span><b> 3(3) </b></span><span> (2022): </span><span><i> JMIRx med, </i></span></li><li><span> "Peer Review of &#8220;Representing Physician Suicide Claims as Nanopublications: Proof-of-Concept Study Creating Claim Networks&#8221;", </span><span><b> 3(3) </b></span><span> (2022): </span><span><i> JMIRx med, </i></span></li><li><span> Chan E., </span><span> "Peer Review of &#8220;Representing Physician Suicide Claims as Nanopublications: Proof-of-Concept Study Creating Claim Networks&#8221;", </span><span><b> 3(3) </b></span><span> (2022): </span><span><i> JMIRx med, </i></span></li><li><span> Ammar A, Evelo C, Willighagen E., </span><span> "FAIR assessment of nanosafety data reusability with community standards.", </span><span><b> 11(1) </b></span><span> (2024): </span><span><i> Scientific data, </i></span><span> 503, </span><span> DOI: 10.1038/s41597-024-03324-x, </span></li><li><span> Akhmetova LA, Kurochkin AS, Frolov AV., </span><span> "Dung-beetles (Coleoptera, Scarabaeidae, Aphodiinae, Scarabaeinae) feeding on faeces of steppe marmots &lt;i&gt;Marmotabobak&lt;/i&gt; (Rodentia, Sciuridae) in Middle Volga territory.", </span><span><b> 12 </b></span><span> (2024): </span><span><i> Biodiversity data journal, </i></span><span> e125090, </span><span> DOI: 10.3897/bdj.12.e125090, </span></li><li><span> Kuhn T, Taelman R, Emonet V, Antonatos H, Soiland-Reyes S, Dumontier M., </span><span> "Semantic micro-contributions with decentralized nanopublication services.", </span><span><b> 7 </b></span><span> (2021): </span><span><i> PeerJ. Computer science, </i></span><span> e387, </span><span> DOI: 10.7717/peerj-cs.387, </span></li><li><span> Vogt L, Kuhn T, Hoehndorf R., </span><span> "Semantic units: organizing knowledge graphs into semantically meaningful units of representation.", </span><span><b> 15(1) </b></span><span> (2024): </span><span><i> Journal of biomedical semantics, </i></span><span> 7, </span><span> DOI: 10.1186/s13326-024-00310-5, </span></li><li><span> Schultes E, Roos M, Bonino da Silva Santos LO, Guizzardi G, Bouwman J, Hankemeier T, Baak A, Mons B., </span><span> "FAIR Digital Twins for Data-Intensive Research.", </span><span><b> 5 </b></span><span> (2022): </span><span><i> Frontiers in big data, </i></span><span> 883341, </span><span> DOI: 10.3389/fdata.2022.883341, </span></li><li><span> Giachelle F, Dosso D, Silvello G., </span><span> "Search, access, and explore life science nanopublications on the Web.", </span><span><b> 7 </b></span><span> (2021): </span><span><i> PeerJ. Computer science, </i></span><span> e335, </span><span> DOI: 10.7717/peerj-cs.335, </span></li><li><span> Ehrhart F, Willighagen EL, Kutmon M, van Hoften M, Curfs LMG, Evelo CT., </span><span> "A resource to explore the discovery of rare diseases and their causative genes.", </span><span><b> 8(1) </b></span><span> (2021): </span><span><i> Scientific data, </i></span><span> 124, </span><span> DOI: 10.1038/s41597-021-00905-y, </span></li><li><span> Dimitrova M, Senderov VE, Georgiev T, Zhelezov G, Penev L., </span><span> "Infrastructure and Population of the OpenBiodiv Biodiversity Knowledge Graph.", </span><span><b> 9 </b></span><span> (2021): </span><span><i> Biodiversity data journal, </i></span><span> e67671, </span><span> DOI: 10.3897/bdj.9.e67671, </span></li><li><span> Martens M, Ammar A, Riutta A, Waagmeester A, Slenter DN, Hanspers K, A Miller R, Digles D, Lopes EN, Ehrhart F, Dupuis LJ, Winckers LA, Coort SL, Willighagen EL, Evelo CT, Pico AR, Kutmon M., </span><span> "WikiPathways: connecting communities.", </span><span><b> 49(D1) </b></span><span> (2021): </span><span><i> Nucleic acids research, </i></span><span> D613-D621, </span><span> DOI: 10.1093/nar/gkaa1024, </span></li><li><span> Willighagen E., </span><span> "Two years of explicit CiTO annotations.", </span><span><b> 15(1) </b></span><span> (2023): </span><span><i> Journal of cheminformatics, </i></span><span> 14, </span><span> DOI: 10.1186/s13321-023-00683-2, </span></li><li><span> Zhang X, Zheng H, Zeng Y, Zou J, Zhao L., </span><span> "Exploring how health-related advertising interference contributes to the development of cyberchondria: A stressor-strain-outcome approach.", </span><span><b> 10 </b></span><span> (2024): </span><span><i> Digital health, </i></span><span> 20552076241233138, </span><span> DOI: 10.1177/20552076241233138, </span></li><li><span> Seneviratne O, Das AK, Chari S, Agu NN, Rashid SM, McCusker J, Franklin JS, Qi M, Bennett KP, Chen CH, Hendler JA, McGuinness DL., </span><span> "Semantically enabling clinical decision support recommendations.", </span><span><b> 14(1) </b></span><span> (2023): </span><span><i> Journal of biomedical semantics, </i></span><span> 8, </span><span> DOI: 10.1186/s13326-023-00285-9, </span></li><li><span> Massey PM, Kearney MD, Hauer MK, Selvan P, Koku E, Leader AE., </span><span> "Dimensions of Misinformation About the HPV Vaccine on Instagram: Content and Network Analysis of Social Media Characteristics.", </span><span><b> 22(12) </b></span><span> (2020): </span><span><i> Journal of medical Internet research, </i></span><span> e21451, </span><span> DOI: 10.2196/21451, </span></li><li><span> Page R., </span><span> "Liberating links between datasets using lightweight data publishing: an example using plant names and the taxonomic literature.", </span><span> (2018): </span><span><i> Biodiversity data journal, </i></span><span> e27539, </span><span> DOI: 10.3897/bdj.6.e27539, </span></li><li><span> Verma S, Bhatia R, Harit S, Batish S., </span><span> "Scholarly knowledge graphs through structuring scholarly communication: a review.", </span><span><b> 9(1) </b></span><span> (2023): </span><span><i> Complex &amp; intelligent systems, </i></span><span> 1059-1095, </span><span> DOI: 10.1007/s40747-022-00806-6, </span></li><li><span> Amith M, Onye C, Ledoux T, Xiong G, Tao C., </span><span> "The ontology of fast food facts: conceptualization of nutritional fast food data for consumers and semantic web applications.", </span><span><b> 21(Suppl 7) </b></span><span> (2021): </span><span><i> BMC medical informatics and decision making, </i></span><span> 275, </span><span> DOI: 10.1186/s12911-021-01636-1, </span></li><li><span> Mina E, Thompson M, Kaliyaperumal R, Zhao J, der Horst vE, Tatum Z, Hettne KM, Schultes EA, Mons B, Roos M., </span><span> "Nanopublications for exposing experimental data in the life-sciences: a Huntington's Disease case study.", </span><span><b> 6 </b></span><span> (2015): </span><span><i> Journal of biomedical semantics, </i></span><span> 5, </span><span> DOI: 10.1186/2041-1480-6-5, </span></li><li><span> Blagec K, Barbosa-Silva A, Ott S, Samwald M., </span><span> "A curated, ontology-based, large-scale knowledge graph of artificial intelligence tasks and benchmarks.", </span><span><b> 9(1) </b></span><span> (2022): </span><span><i> Scientific data, </i></span><span> 322, </span><span> DOI: 10.1038/s41597-022-01435-x, </span></li><li><span> Verspoor K, Kim J, Dumontier M., </span><span> "Interoperability of text corpus annotations with the semantic web", </span><span><b> 9(Suppl 5) </b></span><span> (2015): </span><span><i> BMC proceedings, </i></span><span> A2-A2, </span><span> DOI: 10.1186/1753-6561-9-s5-a2, </span></li><li><span> Amith M, Tao C., </span><span> "Representing vaccine misinformation using ontologies.", </span><span><b> 9(1) </b></span><span> (2018): </span><span><i> Journal of biomedical semantics, </i></span><span> 22, </span><span> DOI: 10.1186/s13326-018-0190-0, </span></li><li><span> Hettne KM, Thompson M, van Haagen HH, van der Horst E, Kaliyaperumal R, Mina E, Tatum Z, Laros JF, van Mulligen EM, Schuemie M, Aten E, Li TS, Bruskiewich R, Good BM, Su AI, Kors JA, den Dunnen J, van Ommen GJ, Roos M, 't Hoen PA, Mons B, Schultes EA., </span><span> "The Implicitome: A Resource for Rationalizing Gene-Disease Associations.", </span><span><b> 11(2) </b></span><span> (2016): </span><span><i> PloS one, </i></span><span> e0149621, </span><span> DOI: 10.1371/journal.pone.0149621, </span></li><li><span> Wilkinson MD, Dumontier M, Sansone SA, Bonino da Silva Santos LO, Prieto M, Batista D, McQuilton P, Kuhn T, Rocca-Serra P, Crosas M, Schultes E., </span><span> "Evaluating FAIR maturity through a scalable, automated, community-governed framework.", </span><span><b> 6(1) </b></span><span> (2019): </span><span><i> Scientific data, </i></span><span> 174, </span><span> DOI: 10.1038/s41597-019-0184-5, </span></li><li><span> Lizio M, Harshbarger J, Shimoji H, Severin J, Kasukawa T, Sahin S, Abugessaisa I, Fukuda S, Hori F, Ishikawa-Kato S, Mungall CJ, Arner E, Baillie JK, Bertin N, Bono H, de Hoon M, Diehl AD, Dimont E, Freeman TC, Fujieda K, Hide W, Kaliyaperumal R, Katayama T, Lassmann T, Meehan TF, Nishikata K, Ono H, Rehli M, Sandelin A, Schultes EA, 't Hoen PA, Tatum Z, Thompson M, Toyoda T, Wright DW, Daub CO, Itoh M, Carninci P, Hayashizaki Y, Forrest AR, Kawaji H, FANTOM consortium., </span><span> "Gateways to the FANTOM5 promoter level mammalian expression atlas.", </span><span><b> 16 </b></span><span> (2015): </span><span><i> Genome biology, </i></span><span> 22, </span><span> DOI: 10.1186/s13059-014-0560-6, </span></li><li><span> Ganapathiraju M, Balakrishnan N., </span><span> "T188. MASSIVE OPEN ONLINE DISCOVERY &#8216;MOOD&#8217; FOR SCHIZOPHRENIA RESEARCH", </span><span><b> 44(Suppl 1) </b></span><span> (2018): </span><span><i> Schizophrenia bulletin, </i></span><span> S189-S189, </span></li><li><span> Gonz&#225;lez-Beltr&#225;n A, Li P, Zhao J, Avila-Garcia MS, Roos M, Thompson M, van der Horst E, Kaliyaperumal R, Luo R, Lee TL, Lam TW, Edmunds SC, Sansone SA, Rocca-Serra P., </span><span> "From Peer-Reviewed to Peer-Reproduced in Scholarly Publishing: The Complementary Roles of Data Models and Workflows in Bioinformatics.", </span><span><b> 10(7) </b></span><span> (2015): </span><span><i> PloS one, </i></span><span> e0127612, </span><span> DOI: 10.1371/journal.pone.0127612, </span></li><li><span> McCusker JP, Dumontier M, Yan R, He S, Dordick JS, McGuinness DL., </span><span> "Finding melanoma drugs through a probabilistic knowledge graph.", </span><span><b> 3 </b></span><span> (2017): </span><span><i> PeerJ. Computer science, </i></span><span> e106, </span><span> DOI: 10.7717/peerj-cs.106, </span></li><li><span> Beck T, Free RC, Thorisson GA, Brookes AJ., </span><span> "Semantically enabling a genome-wide association study database.", </span><span><b> 3(1) </b></span><span> (2012): </span><span><i> Journal of biomedical semantics, </i></span><span> 9, </span><span> DOI: 10.1186/2041-1480-3-9, </span></li><li><span> Masseroli M, Mons B, Bongcam-Rudloff E, Ceri S, Kel A, Rechenmann F, Lisacek F, Romano P., </span><span> "Integrated Bio-Search: challenges and trends for the integration, search and comprehensive processing of biological information.", </span><span><b> 15 Suppl 1 </b></span><span> (2014): </span><span><i> BMC bioinformatics, </i></span><span> S2, </span><span> DOI: 10.1186/1471-2105-15-s1-s2, </span></li><li><span> Slenter DN, Kutmon M, Hanspers K, Riutta A, Windsor J, Nunes N, M&#233;lius J, Cirillo E, Coort SL, Digles D, Ehrhart F, Giesbertz P, Kalafati M, Martens M, Miller R, Nishida K, Rieswijk L, Waagmeester A, Eijssen LMT, Evelo CT, Pico AR, Willighagen EL., </span><span> "WikiPathways: a multifaceted pathway database bridging metabolomics to other omics research.", </span><span><b> 46(D1) </b></span><span> (2018): </span><span><i> Nucleic acids research, </i></span><span> D661-D667, </span><span> DOI: 10.1093/nar/gkx1064, </span></li><li><span> Clark T, Ciccarese PN, Goble CA., </span><span> "Micropublications: a semantic model for claims, evidence, arguments and annotations in biomedical communications.", </span><span><b> 5 </b></span><span> (2014): </span><span><i> Journal of biomedical semantics, </i></span><span> 28, </span><span> DOI: 10.1186/2041-1480-5-28, </span></li><li><span> Lamprecht AL, Palmblad M, Ison J, Schw&#228;mmle V, Al Manir MS, Altintas I, Baker CJO, Ben Hadj Amor A, Capella-Gutierrez S, Charonyktakis P, Crusoe MR, Gil Y, Goble C, Griffin TJ, Groth P, Ienasescu H, Jagtap P, Kala&#353; M, Kasalica V, Khanteymoori A, Kuhn T, Mei H, M&#233;nager H, M&#246;ller S, Richardson RA, Robert V, Soiland-Reyes S, Stevens R, Szaniszlo S, Verberne S, Verhoeven A, Wolstencroft K., </span><span> "Perspectives on automated composition of workflows in the life sciences.", </span><span><b> 10 </b></span><span> (2021): </span><span><i> F1000Research, </i></span><span> 897, </span><span> DOI: 10.12688/f1000research.54159.1, </span></li><li><span> Zhao L, Zhao Y, Wu Y, Ding X, Yu F, Peng K., </span><span> "Impacts of the COVID-19 Pandemic upon Chinese Positive Traits.", </span><span><b> 19(20) </b></span><span> (2022): </span><span><i> International journal of environmental research and public health, </i></span><span> 13490, </span><span> DOI: 10.3390/ijerph192013490, </span></li><li><span> Maldonado JA, Marcos M, Fern&#225;ndez-Breis JT, Parcero E, Bosc&#225; D, Legaz-Garc&#237;a MD, Mart&#237;nez-Salvador B, Robles M., </span><span> "A platform for exploration into chaining of web services for clinical data transformation and reasoning.", </span><span><b> 2016 </b></span><span> (2016): </span><span><i> AMIA ... Annual Symposium proceedings. AMIA Symposium, </i></span><span> 854-863, </span></li><li><span> Neal ML, Gennari JH, Waltemath D, Nickerson DP, K&#246;nig M., </span><span> "Open modeling and exchange (OMEX) metadata specification version 1.0.", </span><span><b> 17(2-3) </b></span><span> (2020): </span><span><i> Journal of integrative bioinformatics, </i></span><span> DOI: 10.1515/jib-2020-0020, </span></li><li><span> Corcho O, Ekaputra FJ, Heibi I, Jonquet C, Micsik A, Peroni S, Storti E., </span><span> "A maturity model for catalogues of semantic artefacts.", </span><span><b> 11(1) </b></span><span> (2024): </span><span><i> Scientific data, </i></span><span> 479, </span><span> DOI: 10.1038/s41597-024-03185-4, </span></li><li><span> McCusker JP, Lebo T, Krauthammer M, McGuinness DL., </span><span> "Next Generation Cancer Data Discovery, Access, and Integration Using Prizms and Nanopublications.", </span><span><b> 7970 </b></span><span> (2013): </span><span><i> Data integration in the life sciences : ... International Workshop, DILS ... : proceedings. DILS (Conference), </i></span><span> 105-112, </span><span> DOI: 10.1007/978-3-642-39437-9_9, </span></li><li><span> Altman M, Cohen PN., </span><span> "The Scholarly Knowledge Ecosystem: Challenges and Opportunities for the Field of Information.", </span><span><b> 6 </b></span><span> (2021): </span><span><i> Frontiers in research metrics and analytics, </i></span><span> 751553, </span><span> DOI: 10.3389/frma.2021.751553, </span></li><li><span> Senderov V, Simov K, Franz N, Stoev P, Catapano T, Agosti D, Sautter G, Morris RA, Penev L., </span><span> "OpenBiodiv-O: ontology of the OpenBiodiv knowledge management system.", </span><span><b> 9(1) </b></span><span> (2018): </span><span><i> Journal of biomedical semantics, </i></span><span> 5, </span><span> DOI: 10.1186/s13326-017-0174-5, </span></li><li><span> Allard PM, Bisson J, Azzollini A, Pauli GF, Cordell GA, Wolfender JL., </span><span> "Pharmacognosy in the digital era: shifting to contextualized metabolomics.", </span><span><b> 54 </b></span><span> (2018): </span><span><i> Current opinion in biotechnology, </i></span><span> 57-64, </span><span> DOI: 10.1016/j.copbio.2018.02.010, </span></li><li><span> Raciti D, Yook K, Harris TW, Schedl T, Sternberg PW., </span><span> "Micropublication: incentivizing community curation and placing unpublished data into the public domain.", </span><span><b> 2018 </b></span><span> (2018): </span><span><i> Database : the journal of biological databases and curation, </i></span><span> DOI: 10.1093/database/bay013, </span></li><li><span> Ciccarese P, Soiland-Reyes S, Belhajjame K, Gray AJ, Goble C, Clark T., </span><span> "PAV ontology: provenance, authoring and versioning.", </span><span><b> 4(1) </b></span><span> (2013): </span><span><i> Journal of biomedical semantics, </i></span><span> 37, </span><span> DOI: 10.1186/2041-1480-4-37, </span></li><li><span> Samuel S, Mietchen D., </span><span> "Computational reproducibility of Jupyter notebooks from biomedical publications.", </span><span><b> 13 </b></span><span> (2024): </span><span><i> GigaScience, </i></span><span> giad113, </span><span> DOI: 10.1093/gigascience/giad113, </span></li><li><span> Pi&#241;ero J, Bravo &#192;, Queralt-Rosinach N, Guti&#233;rrez-Sacrist&#225;n A, Deu-Pons J, Centeno E, Garc&#237;a-Garc&#237;a J, Sanz F, Furlong LI., </span><span> "DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants.", </span><span><b> 45(D1) </b></span><span> (2017): </span><span><i> Nucleic acids research, </i></span><span> D833-D839, </span><span> DOI: 10.1093/nar/gkw943, </span></li><li><span> Groom Q, Br&#228;uchler C, Cubey RWN, Dillen M, Huybrechts P, Kearney N, Klazenga N, Leachman S, Paul DL, Rogers H, Santos J, Shorthouse DP, Vaughan A, von Mering S, Haston EM., </span><span> "The disambiguation of people names in biological collections.", </span><span><b> 10 </b></span><span> (2022): </span><span><i> Biodiversity data journal, </i></span><span> e86089, </span><span> DOI: 10.3897/bdj.10.e86089, </span></li><li><span> Romano P, Cannata N., </span><span> "NETTAB 2013: Semantic, social, and mobile applications for bioinformatics and biomedical laboratories.", </span><span><b> 15 Suppl 14 </b></span><span> (2014): </span><span><i> BMC bioinformatics, </i></span><span> S1, </span><span> DOI: 10.1186/1471-2105-15-s14-s1, </span></li><li><span> Kilicoglu H., </span><span> "Biomedical text mining for research rigor and integrity: tasks, challenges, directions.", </span><span><b> 19(6) </b></span><span> (2018): </span><span><i> Briefings in bioinformatics, </i></span><span> 1400-1414, </span><span> DOI: 10.1093/bib/bbx057, </span></li><li><span> Prieto M, Deus H, de Waard A, Schultes E, Garc&#237;a-Jim&#233;nez B, Wilkinson MD., </span><span> "Data-driven classification of the certainty of scholarly assertions.", </span><span><b> 8 </b></span><span> (2020): </span><span><i> PeerJ, </i></span><span> e8871, </span><span> DOI: 10.7717/peerj.8871, </span></li><li><span> Ammar A, Bonaretti S, Winckers L, Quik J, Bakker M, Maier D, Lynch I, van Rijn J, Willighagen E., </span><span> "A Semi-Automated Workflow for FAIR Maturity Indicators in the Life Sciences.", </span><span><b> 10(10) </b></span><span> (2020): </span><span><i> Nanomaterials (Basel, Switzerland), </i></span><span> E2068, </span><span> DOI: 10.3390/nano10102068, </span></li><li><span> Lizio M, Harshbarger J, Abugessaisa I, Noguchi S, Kondo A, Severin J, Mungall C, Arenillas D, Mathelier A, Medvedeva YA, Lennartsson A, Drabl&#248;s F, Ramilowski JA, Rackham O, Gough J, Andersson R, Sandelin A, Ienasescu H, Ono H, Bono H, Hayashizaki Y, Carninci P, Forrest AR, Kasukawa T, Kawaji H., </span><span> "Update of the FANTOM web resource: high resolution transcriptome of diverse cell types in mammals.", </span><span><b> 45(D1) </b></span><span> (2017): </span><span><i> Nucleic acids research, </i></span><span> D737-D743, </span><span> DOI: 10.1093/nar/gkw995, </span></li><li><span> Barros M, Couto FM., </span><span> "Knowledge Representation and Management: a Linked Data Perspective.", </span><span> (2016): </span><span><i> Yearbook of medical informatics, </i></span><span> 178-183, </span><span> DOI: 10.15265/iy-2016-022, </span></li><li><span> Townend GS, Ehrhart F, van Kranen HJ, Wilkinson M, Jacobsen A, Roos M, Willighagen EL, van Enckevort D, Evelo CT, Curfs LMG., </span><span> "MECP2 variation in Rett syndrome-An overview of current coverage of genetic and phenotype data within existing databases.", </span><span><b> 39(7) </b></span><span> (2018): </span><span><i> Human mutation, </i></span><span> 914-924, </span><span> DOI: 10.1002/humu.23542, </span></li><li><span> Sarntivijai S, Diehl AD, He Y., </span><span> "Cells in experimental life sciences - challenges and solution to the rapid evolution of knowledge.", </span><span><b> 18(Suppl 17) </b></span><span> (2017): </span><span><i> BMC bioinformatics, </i></span><span> 560, </span><span> DOI: 10.1186/s12859-017-1976-2, </span></li><li><span> Markel S., </span><span> "BioLINK special interest group session on the future of scientific publishing.", </span><span><b> 5(5) </b></span><span> (2009): </span><span><i> PLoS computational biology, </i></span><span> e1000398, </span><span> DOI: 10.1371/journal.pcbi.1000398, </span></li><li><span> Callahan A, Dumontier M, Shah NH., </span><span> "HyQue: evaluating hypotheses using Semantic Web technologies.", </span><span><b> 2 Suppl 2 </b></span><span> (2011): </span><span><i> Journal of biomedical semantics, </i></span><span> S3, </span><span> DOI: 10.1186/2041-1480-2-s2-s3, </span></li><li><span> Buneman P, Christie G, Davies JA, Dimitrellou R, Harding SD, Pawson AJ, Sharman JL, Wu Y., </span><span> "Why data citation isn't working, and what to do about it.", </span><span><b> 2020 </b></span><span> (2020): </span><span><i> Database : the journal of biological databases and curation, </i></span><span> baaa022, </span><span> DOI: 10.1093/databa/baaa022, </span></li><li><span> Waterhouse RM, Adam-Blondon AF, Balech B, Barta E, Ying Shi Chua P, Di Cola V, Heil KF, Hughes GM, Jermiin LS, Kala&#353; M, Lanfear J, Pafilis E, Palagi PM, Papageorgiou AC, Paup&#233;rio J, Psomopoulos F, Raes N, Burgin J, Gabald&#243;n T., </span><span> "The ELIXIR Biodiversity Community: Understanding short- and long-term changes in biodiversity.", </span><span><b> 12 </b></span><span> (2023): </span><span><i> F1000Research, </i></span><span> ELIXIR-499, </span><span> DOI: 10.12688/f1000research.133724.2, </span></li><li><span> Sarntivijai S, Vasant D, Jupp S, Saunders G, Bento AP, Gonzalez D, Betts J, Hasan S, Koscielny G, Dunham I, Parkinson H, Malone J., </span><span> "Linking rare and common disease: mapping clinical disease-phenotypes to ontologies in therapeutic target validation.", </span><span><b> 7 </b></span><span> (2016): </span><span><i> Journal of biomedical semantics, </i></span><span> 8, </span><span> DOI: 10.1186/s13326-016-0051-7, </span></li><li><span> Hettne KM, Dharuri H, Zhao J, Wolstencroft K, Belhajjame K, Soiland-Reyes S, Mina E, Thompson M, Cruickshank D, Verdes-Montenegro L, Garrido J, de Roure D, Corcho O, Klyne G, van Schouwen R, 't Hoen PA, Bechhofer S, Goble C, Roos M., </span><span> "Structuring research methods and data with the research object model: genomics workflows as a case study.", </span><span><b> 5(1) </b></span><span> (2014): </span><span><i> Journal of biomedical semantics, </i></span><span> 41, </span><span> DOI: 10.1186/2041-1480-5-41, </span></li><li><span> Ammar N, Bailey JE, Davis RL, Shaban-Nejad A., </span><span> "Using a Personal Health Library-Enabled mHealth Recommender System for Self-Management of Diabetes Among Underserved Populations: Use Case for Knowledge Graphs and Linked Data.", </span><span><b> 5(3) </b></span><span> (2021): </span><span><i> JMIR formative research, </i></span><span> e24738, </span><span> DOI: 10.2196/24738, </span></li><li><span> Oetting WS, Robinson PN, Greenblatt MS, Cotton RG, Beck T, Carey JC, Doelken SC, Girdea M, Groza T, Hamilton CM, Hamosh A, Kerner B, MacArthur JA, Maglott DR, Mons B, Rehm HL, Schofield PN, Searle BA, Smedley D, Smith CL, Bernstein IT, Zankl A, Zhao EY., </span><span> "Getting ready for the Human Phenome Project: the 2012 forum of the Human Variome Project.", </span><span><b> 34(4) </b></span><span> (2013): </span><span><i> Human mutation, </i></span><span> 661-666, </span><span> DOI: 10.1002/humu.22293, </span></li><li><span> Holinski A, Burke ML, Morgan SL, McQuilton P, Palagi PM., </span><span> "Biocuration - mapping resources and needs.", </span><span><b> 9 </b></span><span> (2020): </span><span><i> F1000Research, </i></span><span> ELIXIR-1094, </span><span> DOI: 10.12688/f1000research.25413.2, </span></li><li><span> Cui H, Ford B, Starr J, Reznicek A, Zhang L, Macklin JA., </span><span> "Authors' attitude toward adopting a new workflow to improve the computability of phenotype publications.", </span><span><b> 2022 </b></span><span> (2022): </span><span><i> Database : the journal of biological databases and curation, </i></span><span> baac001, </span><span> DOI: 10.1093/database/baac001, </span></li><li><span> Koscielny G, An P, Carvalho-Silva D, Cham JA, Fumis L, Gasparyan R, Hasan S, Karamanis N, Maguire M, Papa E, Pierleoni A, Pignatelli M, Platt T, Rowland F, Wankar P, Bento AP, Burdett T, Fabregat A, Forbes S, Gaulton A, Gonzalez CY, Hermjakob H, Hersey A, Jupe S, Kafkas &#350;, Keays M, Leroy C, Lopez FJ, Magarinos MP, Malone J, McEntyre J, Munoz-Pomer Fuentes A, O'Donovan C, Papatheodorou I, Parkinson H, Palka B, Paschall J, Petryszak R, Pratanwanich N, Sarntivijal S, Saunders G, Sidiropoulos K, Smith T, Sondka Z, Stegle O, Tang YA, Turner E, Vaughan B, Vrousgou O, Watkins X, Martin MJ, Sanseau P, Vamathevan J, Birney E, Barrett J, Dunham I., </span><span> "Open Targets: a platform for therapeutic target identification and validation.", </span><span><b> 45(D1) </b></span><span> (2017): </span><span><i> Nucleic acids research, </i></span><span> D985-D994, </span><span> DOI: 10.1093/nar/gkw1055, </span></li><li><span> Rinaldi A., </span><span> "For I dipped into the future.", </span><span><b> 11(5) </b></span><span> (2010): </span><span><i> EMBO reports, </i></span><span> 345-349, </span><span> DOI: 10.1038/embor.2010.57, </span></li><li><span> Burns GA, Turner JA., </span><span> "Modeling functional Magnetic Resonance Imaging (fMRI) experimental variables in the Ontology of Experimental Variables and Values (OoEVV).", </span><span><b> 82 </b></span><span> (2013): </span><span><i> NeuroImage, </i></span><span> 662-670, </span><span> DOI: 10.1016/j.neuroimage.2013.05.024, </span></li><li><span> Goldmann D, Zdrazil B, Digles D, Ecker GF., </span><span> "Empowering pharmacoinformatics by linked life science data.", </span><span><b> 31(3) </b></span><span> (2017): </span><span><i> Journal of computer-aided molecular design, </i></span><span> 319-328, </span><span> DOI: 10.1007/s10822-016-9990-4, </span></li><li><span> K&#246;hler CA, Ulianych D, Gr&#252;n S, Decker S, Denker M., </span><span> "Facilitating the Sharing of Electrophysiology Data Analysis Results Through In-Depth Provenance Capture.", </span><span><b> 11(6) </b></span><span> (2024): </span><span><i> eNeuro, </i></span><span> ENEURO.0476-23.2024, </span><span> DOI: 10.1523/eneuro.0476-23.2024, </span></li><li><span> Livingston KM, Bada M, Hunter LE, Verspoor K., </span><span> "Representing annotation compositionality and provenance for the Semantic Web.", </span><span><b> 4 </b></span><span> (2013): </span><span><i> Journal of biomedical semantics, </i></span><span> 38, </span><span> DOI: 10.1186/2041-1480-4-38, </span></li><li><span> Masara B, van der Poll JA, Maaza M., </span><span> "A nanotechnology-foresight perspective of South Africa.", </span><span><b> 23(4) </b></span><span> (2021): </span><span><i> Journal of nanoparticle research : an interdisciplinary forum for nanoscale science and technology, </i></span><span> 92, </span><span> DOI: 10.1007/s11051-021-05193-6, </span></li><li><span> Tiddi I, Balliet D, ten Teije A., </span><span> "Fostering Scientific Meta-analyses with Knowledge Graphs: A Case-Study", </span><span><b> 12123 </b></span><span> (2020): </span><span><i> The Semantic Web17th International Conference, ESWC 2020, Heraklion, Crete, Greece, May 31&#8211;June 4, 2020, Proceedings, </i></span><span> 287-303, </span></li><li><span> Garcia A, Lopez F, Garcia L, Giraldo O, Bucheli V, Dumontier M., </span><span> "Biotea: semantics for Pubmed Central.", </span><span><b> 6 </b></span><span> (2018): </span><span><i> PeerJ, </i></span><span> e4201, </span><span> DOI: 10.7717/peerj.4201, </span></li><li><span> Shotton D, Portwin K, Klyne G, Miles A., </span><span> "Adventures in semantic publishing: exemplar semantic enhancements of a research article.", </span><span><b> 5(4) </b></span><span> (2009): </span><span><i> PLoS computational biology, </i></span><span> e1000361, </span><span> DOI: 10.1371/journal.pcbi.1000361, </span></li><li><span> Saqi M, Lysenko A, Guo YK, Tsunoda T, Auffray C., </span><span> "Navigating the disease landscape: knowledge representations for contextualizing molecular signatures.", </span><span><b> 20(2) </b></span><span> (2019): </span><span><i> Briefings in bioinformatics, </i></span><span> 609-623, </span><span> DOI: 10.1093/bib/bby025, </span></li><li><span> B&#246;lling C, Weidlich M, Holzh&#252;tter HG., </span><span> "SEE: structured representation of scientific evidence in the biomedical domain using Semantic Web techniques.", </span><span><b> 5(Suppl 1 Proceedings of the Bio-Ontologies Spec Interest) </b></span><span> (2014): </span><span><i> Journal of biomedical semantics, </i></span><span> S1, </span><span> DOI: 10.1186/2041-1480-5-s1-s1, </span></li><li><span> Thompson BA, Spurdle AB, Plazzer JP, Greenblatt MS, Akagi K, Al-Mulla F, Bapat B, Bernstein I, Capell&#225; G, den Dunnen JT, du Sart D, Fabre A, Farrell MP, Farrington SM, Frayling IM, Frebourg T, Goldgar DE, Heinen CD, Holinski-Feder E, Kohonen-Corish M, Robinson KL, Leung SY, Martins A, Moller P, Morak M, Nystrom M, Peltomaki P, Pineda M, Qi M, Ramesar R, Rasmussen LJ, Royer-Pokora B, Scott RJ, Sijmons R, Tavtigian SV, Tops CM, Weber T, Wijnen J, Woods MO, Macrae F, Genuardi M., </span><span> "Application of a 5-tiered scheme for standardized classification of 2,360 unique mismatch repair gene variants in the InSiGHT locus-specific database.", </span><span><b> 46(2) </b></span><span> (2014): </span><span><i> Nature genetics, </i></span><span> 107-115, </span><span> DOI: 10.1038/ng.2854, </span></li><li><span> Rebholz-Schuhmann D, Nenadic G., </span><span> "Biomedical semantics: the hub for biomedical research 2.0.", </span><span><b> 1(1) </b></span><span> (2010): </span><span><i> Journal of biomedical semantics, </i></span><span> 1, </span><span> DOI: 10.1186/2041-1480-1-1, </span></li><li><span> Vos RA, Katayama T, Mishima H, Kawano S, Kawashima S, Kim JD, Moriya Y, Tokimatsu T, Yamaguchi A, Yamamoto Y, Wu H, Amstutz P, Antezana E, Aoki NP, Arakawa K, Bolleman JT, Bolton E, Bonnal RJP, Bono H, Burger K, Chiba H, Cohen KB, Deutsch EW, Fern&#225;ndez-Breis JT, Fu G, Fujisawa T, Fukushima A, Garc&#237;a A, Goto N, Groza T, Hercus C, Hoehndorf R, Itaya K, Juty N, Kawashima T, Kim JH, Kinjo AR, Kotera M, Kozaki K, Kumagai S, Kushida T, L&#252;tteke T, Matsubara M, Miyamoto J, Mohsen A, Mori H, Naito Y, Nakazato T, Nguyen-Xuan J, Nishida K, Nishida N, Nishide H, Ogishima S, Ohta T, Okuda S, Paten B, Perret JL, Prathipati P, Prins P, Queralt-Rosinach N, Shinmachi D, Suzuki S, Tabata T, Takatsuki T, Taylor K, Thompson M, Uchiyama I, Vieira B, Wei CH, Wilkinson M, Yamada I, Yamanaka R, Yoshitake K, Yoshizawa AC, Dumontier M, Kosaki K, Takagi T., </span><span> "BioHackathon 2015: Semantics of data for life sciences and reproducible research.", </span><span><b> 9 </b></span><span> (2020): </span><span><i> F1000Research, </i></span><span> 136, </span><span> DOI: 10.12688/f1000research.18236.1, </span></li><li><span> Neal ML, K&#246;nig M, Nickerson D, M&#305;s&#305;rl&#305; G, Kalbasi R, Dr&#228;ger A, Atalag K, Chelliah V, Cooling MT, Cook DL, Crook S, de Alba M, Friedman SH, Garny A, Gennari JH, Gleeson P, Golebiewski M, Hucka M, Juty N, Myers C, Olivier BG, Sauro HM, Scharm M, Snoep JL, Tour&#233; V, Wipat A, Wolkenhauer O, Waltemath D., </span><span> "Harmonizing semantic annotations for computational models in biology.", </span><span><b> 20(2) </b></span><span> (2019): </span><span><i> Briefings in bioinformatics, </i></span><span> 540-550, </span><span> DOI: 10.1093/bib/bby087, </span></li><li><span> Dumontier M, Baker CJ, Baran J, Callahan A, Chepelev L, Cruz-Toledo J, Del Rio NR, Duck G, Furlong LI, Keath N, Klassen D, McCusker JP, Queralt-Rosinach N, Samwald M, Villanueva-Rosales N, Wilkinson MD, Hoehndorf R., </span><span> "The Semanticscience Integrated Ontology (SIO) for biomedical research and knowledge discovery.", </span><span><b> 5(1) </b></span><span> (2014): </span><span><i> Journal of biomedical semantics, </i></span><span> 14, </span><span> DOI: 10.1186/2041-1480-5-14, </span></li><li><span> Ekins S, Clark AM, Williams AJ., </span><span> "Open Drug Discovery Teams: A Chemistry Mobile App for Collaboration.", </span><span><b> 31(8) </b></span><span> (2012): </span><span><i> Molecular informatics, </i></span><span> 585-597, </span><span> DOI: 10.1002/minf.201200034, </span></li><li><span> Russ TA, Ramakrishnan C, Hovy EH, Bota M, Burns GA., </span><span> "Knowledge engineering tools for reasoning with scientific observations and interpretations: a neural connectivity use case.", </span><span><b> 12 </b></span><span> (2011): </span><span><i> BMC bioinformatics, </i></span><span> 351, </span><span> DOI: 10.1186/1471-2105-12-351, </span></li><li><span> Read KB, Sheehan JR, Huerta MF, Knecht LS, Mork JG, Humphreys BL, NIH Big Data Annotator Group., </span><span> "Sizing the Problem of Improving Discovery and Access to NIH-Funded Data: A Preliminary Study.", </span><span><b> 10(7) </b></span><span> (2015): </span><span><i> PloS one, </i></span><span> e0132735, </span><span> DOI: 10.1371/journal.pone.0132735, </span></li><li><span> Pang C, Sollie A, Sijtsma A, Hendriksen D, Charbon B, de Haan M, de Boer T, Kelpin F, Jetten J, van der Velde JK, Smidt N, Sijmons R, Hillege H, Swertz MA., </span><span> "SORTA: a system for ontology-based re-coding and technical annotation of biomedical phenotype data.", </span><span><b> 2015 </b></span><span> (2015): </span><span><i> Database : the journal of biological databases and curation, </i></span><span> bav089, </span><span> DOI: 10.1093/database/bav089, </span></li><li><span> Frey JG, Bird CL., </span><span> "Cheminformatics and the Semantic Web: adding value with linked data and enhanced provenance.", </span><span><b> 3(5) </b></span><span> (2013): </span><span><i> Wiley interdisciplinary reviews. Computational molecular science, </i></span><span> 465-481, </span><span> DOI: 10.1002/wcms.1127, </span></li><li><span> Riggs ER, Jackson L, Miller DT, Van Vooren S., </span><span> "Phenotypic information in genomic variant databases enhances clinical care and research: the International Standards for Cytogenomic Arrays Consortium experience.", </span><span><b> 33(5) </b></span><span> (2012): </span><span><i> Human mutation, </i></span><span> 787-796, </span><span> DOI: 10.1002/humu.22052, </span></li><li><span> Cases M, Furlong LI, Albanell J, Altman RB, Bellazzi R, Boyer S, Brand A, Brookes AJ, Brunak S, Clark TW, Gea J, Ghazal P, Graf N, Guig&#243; R, Klein TE, L&#243;pez-Bigas N, Maojo V, Mons B, Musen M, Oliveira JL, Rowe A, Ruch P, Shabo A, Shortliffe EH, Valencia A, van der Lei J, Mayer MA, Sanz F., </span><span> "Improving data and knowledge management to better integrate health care and research.", </span><span><b> 274(4) </b></span><span> (2013): </span><span><i> Journal of internal medicine, </i></span><span> 321-328, </span><span> DOI: 10.1111/joim.12105, </span></li><li><span> Harland L, Larminie C, Sansone SA, Popa S, Marshall MS, Braxenthaler M, Cantor M, Filsell W, Forster MJ, Huang E, Matern A, Musen M, Saric J, Slater T, Wilson J, Lynch N, Wise J, Dix I., </span><span> "Empowering industrial research with shared biomedical vocabularies.", </span><span><b> 16(21-22) </b></span><span> (2011): </span><span><i> Drug discovery today, </i></span><span> 940-947, </span><span> DOI: 10.1016/j.drudis.2011.09.013, </span></li><li><span> Murray-Rust P, Rzepa HS., </span><span> "Semantic physical science.", </span><span><b> 4(1) </b></span><span> (2012): </span><span><i> Journal of cheminformatics, </i></span><span> 14, </span><span> DOI: 10.1186/1758-2946-4-14, </span></li><li><span> Attwood TK, Kell DB, McDermott P, Marsh J, Pettifer SR, Thorne D., </span><span> "Calling International Rescue: knowledge lost in literature and data landslide!", </span><span><b> 424(3) </b></span><span> (2009): </span><span><i> The Biochemical journal, </i></span><span> 317-333, </span><span> DOI: 10.1042/bj20091474, </span></li><li><span> Hoehndorf R, Dumontier M, Gkoutos GV., </span><span> "Evaluation of research in biomedical ontologies.", </span><span><b> 14(6) </b></span><span> (2013): </span><span><i> Briefings in bioinformatics, </i></span><span> 696-712, </span><span> DOI: 10.1093/bib/bbs053, </span></li><li><span> Stobbe MD, Swertz MA, Thiele I, Rengaw T, van Kampen AH, Moerland PD., </span><span> "Consensus and conflict cards for metabolic pathway databases.", </span><span><b> 7 </b></span><span> (2013): </span><span><i> BMC systems biology, </i></span><span> 50, </span><span> DOI: 10.1186/1752-0509-7-50, </span></li><li><span> Keator DB, Helmer K, Steffener J, Turner JA, Van Erp TG, Gadde S, Ashish N, Burns GA, Nichols BN., </span><span> "Towards structured sharing of raw and derived neuroimaging data across existing resources.", </span><span><b> 82 </b></span><span> (2013): </span><span><i> NeuroImage, </i></span><span> 647-661, </span><span> DOI: 10.1016/j.neuroimage.2013.05.094, </span></li></ul></body></html>